serverless.yml

```diff
@@ -102,6 +102,23 @@ functions:
           enabled: true
     vpc: ${self:custom.vpc}
 
+  ControlDbUtilization:
+    handler: src.handler.control_db_utilization
+    role: arn:aws:iam::${self:custom.accountNumber}:role/csr-Lambda-Role
+    environment:
+      AWS_DEPLOYMENT_REGION: ${self:provider.region}
+      LOG_LEVEL: INFO
+      STAGE: ${self:provider.stage}
+      MAX_RETRIES: 6
+    events:
+      - cloudwatchEvent:
+          event:
+            source:
+              - 'aws.cloudwatch'
+            detail:
+              name: ["AQ-SCASG-TEST-203S5-AQTSTEST-A-SDA"]
```

Comment:
Isn't this one of the AQTS's alarms? I don't believe it's the errorHandler's.

Reply:
I'm doing this for testing because right now our error handler is not firing.  I created this review with a cron checking the db utilization, but Jim asked me to hook up to cloudwatch.  Disable triggers on error-handler alarm, enable on OK.  I will switch the name to error handler shortly.

Reply:
Fixed.